lib/core/elastic_connector_actions.rb

```diff
@@ -44,6 +43,14 @@ def connectors_meta
       end
 
       def native_connectors
+        connectors({ term: { is_native: true } })
+      end
+
+      def crawler_connectors
+        connectors({ term: { service_type: Utility::Constants::CRAWLER_SERVICE_TYPE } })
```

Comment:
Do we want to also add a term to ignore crawler in all other places? Probably not a big deal to not do this, just thinking out loud

Reply:
Yes, I totally agree.

Reply:
I'll add it in separate PR